scripts/install-mender.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+
+set -e
+
+CHANNEL="stable"
+
+DEFAULT_COMPONENTS="mender-client"
+COMPONENTS="$DEFAULT_COMPONENTS mender-shell"
+SELECTED_COMPONENTS=$DEFAULT_COMPONENTS
+
+export DEBIAN_FRONTEND=noninteractive
+
+banner (){
+    echo "
+                          _
+ _ __ ___   ___ _ __   __| | ___ _ __ 
+| '_ \` _ \ / _ \ '_ \ / _\` |/ _ \ '__|
+| | | | | |  __/ | | | (_| |  __/ |   
+|_| |_| |_|\___|_| |_|\__,_|\___|_|   
+
+Running the Mender installation script.
+--
+"
+                                      
+}
+
+usage() {
+    echo "usage: install-mender.sh [-h] [-c channel] [component...]"
+    echo "  -h             print this help"
+    echo "  -c             channel: stable(default)|experimental"
```

Comment:
One requires an argument, the other none. This is confusing to me, I suggest:
```suggestion
    echo "  -h                    print this help"
    echo "  -c CHANNEL  channel: stable(default)|experimental"
```